### DISCUSSION TITLE
Working sample app?

### DISCUSSION THREAD

**Discussion by eanagel** [isar/isar#604] [Help] [answered]
I'm currently researching moving to a new data store and isar looks really promising, but I've had trouble finding a working sample app that I can explore. I tried out the "sample" app repo and it appears badly out of date (see https://github.com/isar/samples/issues/8). Any suggestions?

**Comment by simc** [comment, marked as answer]
I hear you! There are plans to build more sample apps. I'm also hoping for some community contributions.

In the meantime check out this video https://youtu.be/CwC9-a9hJv4
